sections/api/primary/styled-component.js

```diff
@@ -17,30 +17,6 @@ const StyledComponent = () => md`
   next to the styles defined by the styled call.
   (e.g. \`<MyStyledComp className="bootstrap__btn" />\`)
 
-  #### .extend (_deprecated_)
```

Comment:
I see this is removed now, but we probably want to keep it around, right? Otherwise our docs aren't valid for old versions and it might become confusing to some :)